test/GRAPHFILES

```diff
@@ -327,3 +327,5 @@ statements/lydia/moduleVersusFunction.graph
 # suite: Samples (bogus)
 Samples/Performance/foo.graph
 Samples/Performance/cleanCopy/foobar.graph
+# suite: Lifetime Management
```

Comment:
Can you rename this "Memory allocation" and put it after the "Memory tracking" suite?

Reply:
Done.